src/java.desktop/share/classes/java/awt/AWTPermission.java

```diff
@@ -34,133 +34,9 @@
- *     display enable malicious code to snoop on the activities of the user.
- *   <tr>
- *     <th scope="row">replaceKeyboardFocusManager
- *     <td>Sets the {@code KeyboardFocusManager} for a particular thread.
- *     <td>When {@code SecurityManager} is installed, the invoking thread must
- *     be granted this permission in order to replace the current
- *     {@code KeyboardFocusManager}. If permission is not granted, a
- *     {@code SecurityException} will be thrown.
- *   <tr>
- *     <th scope="row">setAppletStub
- *     <td>Setting the stub which implements Applet container services
- *     <td>Malicious code could set an applet's stub and result in unexpected
- *     behavior or denial of service to an applet.
- *   <tr>
- *     <th scope="row">setWindowAlwaysOnTop
- *     <td>Setting always-on-top property of the window:
- *     {@link Window#setAlwaysOnTop}
- *     <td>The malicious window might make itself look and behave like a real
- *     full desktop, so that information entered by the unsuspecting user is
- *     captured and subsequently misused
- *   <tr>
- *     <th scope="row">showWindowWithoutWarningBanner
- *     <td>Display of a window without also displaying a banner warning that the
- *     window was created by an applet
- *     <td>Without this warning, an applet may pop up windows without the user
- *     knowing that they belong to an applet. Since users may make
- *     security-sensitive decisions based on whether or not the window belongs
- *     to an applet (entering a username and password into a dialog box, for
- *     example), disabling this warning banner may allow applets to trick the
- *     user into entering such information.
- *   <tr>
- *     <th scope="row">toolkitModality
- *     <td>Creating {@link Dialog.ModalityType#TOOLKIT_MODAL TOOLKIT_MODAL}
- *     dialogs and setting the
- *     {@link Dialog.ModalExclusionType#TOOLKIT_EXCLUDE TOOLKIT_EXCLUDE} window
- *     property.
- *     <td>When a toolkit-modal dialog is shown from an applet, it blocks all
- *     other applets in the browser. When launching applications from Java Web
- *     Start, its windows (such as the security dialog) may also be blocked by
- *     toolkit-modal dialogs, shown from these applications.
- *   <tr>
- *     <th scope="row">watchMousePointer
- *     <td>Getting the information about the mouse pointer position at any time
- *     <td>Constantly watching the mouse pointer, an applet can make guesses
- *     about what the user is doing, i.e. moving the mouse to the lower left
- *     corner of the screen most likely means that the user is about to launch
- *     an application. If a virtual keypad is used so that keyboard is emulated
- *     using the mouse, an applet may guess what is being typed.
- * </tbody>
- * </table>
+ * @apiNote
+ * This permission cannot be used for controlling access to resources anymore
+ * as the Security Manager is no longer supported.
```

Comment:
JEP 486 lists deprecating the permission classes as future work. It would be very disruptive to do this now, but will be a lot easier once the JEP is integrated and the post-JEP tasks to remove the hundreds of usages in the JDK.